### PR TITLE
Changing input_w_feedlib.json to work with master

### PR DESCRIPTION
**Pull request by militsasotirova**
These edits allow input_w_feedlib.json to work with master.

### CODE REVIEW

**Review by mmotew — APPROVED** [review]

**Comment by mmotew** [comment]
If you need someone to merge this after the second review comes in let me know and I can do it.

**Review by wmaxdonovan — APPROVED** [review]
This looks great– Jacob and I have done the same thing in our branch for both this JSON file and barnyard.json. They should all run in Master now.

**Pull request merged**
